packages/editor/src/components/rich-text/tinymce.js

```diff
@@ -337,6 +338,43 @@ export default class TinyMCE extends Component {
 		}
 	}
 
+	/**
+	 * Replaces non breaking spaces inserted by TinyMCE on the space key by
+	 * a normal space.
+	 *
+	 * @param  {SyntheticEvent} event Synthetic key up event.
+	 */
+	onKeyUp( event ) {
```

Comment:
What's the flow by which this actually makes its way into the persisted value, if we're just modifying the DOM directly? I suppose it relies on some subsequent keypress to reflect the DOM revision which had been made on the previous keypress?

Reply:
For some reason TinyMCE also intentionally or not prevents or doesn't trigger an `input` event for this particular space, so the non breaking space is not added to state. The space we added here will be synced on the next input event. Not ideal but I think it's fine as a temporary solution.